### ISSUE TITLE
WHY c3d_resnet18_ucf101 BatchNormalization layers have 3 param?

### ISSUE THREAD

**Issue by bityangke** [facebook/C3D#226]
In https://github.com/facebook/C3D/blob/master/C3D-v1.1/examples/c3d_ucf101_feature_extraction/c3d_resnet18_ucf101_feature_extraction.prototxt
There are three sets of param in BatchNormalization layers. Their shapes are like this:
(channel, ), (channel, ), (1,)

**Comment by dutran** [comment]
@bityangke You may check it out at https://github.com/facebook/C3D/blob/master/C3D-v1.1/src/caffe/layers/batch_norm_layer.cpp#L25-L42
